optimum/utils/normalized_config.py

```diff
@@ -30,7 +30,7 @@ class NormalizedConfig:
     """
 
     def __init__(self, config: Union[PretrainedConfig, Dict], allow_new: bool = False, **kwargs):
-        self.config = config if isinstance(config, PretrainedConfig) else PretrainedConfig.from_dict(config)
+        self.config = config
```

Comment:
Why?

Reply:
We can't create an instance of `PretrainedConfig` for certain `diffusers` config since the addition of https://github.com/huggingface/diffusers/blob/8d8b4311b9e19c3621dbd69e6690ee7be5702f65/src/diffusers/configuration_utils.py#L553 [`_use_default_values`](https://github.com/huggingface/diffusers/blob/8d8b4311b9e19c3621dbd69e6690ee7be5702f65/src/diffusers/configuration_utils.py#L610) which is a set (so not JSON serializable). We could easily make it compatible, but we shouldn't even create an instance of `PretrainedConfig` in the first place IMO, it's not clear to me why `transformers` should be a hard dependency when a user wants to export a `diffusers` model

Reply:
Fixed in https://github.com/huggingface/diffusers/pull/4040, still think we don't need to create a `PretrainedConfig` instance